Core/src/Surfaces/LineSurface.cpp

```diff
@@ -110,16 +102,15 @@ std::string Acts::LineSurface::name() const {
 Acts::RotationMatrix3 Acts::LineSurface::referenceFrame(
```

Comment:
I think we need to document more thoroughly what this ends up calculating.

- y -> line direction
- x -> perp line direction and particle direction
- z -> perp x&y

- None of the axes is parallel to the particle direction in the end
- z points in the direction of the PCA (due to perpendicularities).

Reply:
I think it boils down to what is `d0` and `z0` and in which order do we store them. I always have to guess but I think it is `d0,z0` which corresponds to `x,y` and then z is the "surface normal" so just the cross product of `x` and `y` in the global frame.